### PR TITLE
fix(queries): align descriptionText to similar queries across different platforms

### PR DESCRIPTION
**Pull request by roi-orca**
align descriptionText to similar queries across different platforms

**Proposed Changes**
- align descriptionText to similar queries across different platforms which share generic information and not platform specific

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Comment by roi-orca** [comment]
I did force push to the branch to fix tests, next time I will add commits instead, @rafaela-soares can you please approve the running workflows again? thanks!

**Comment by rafaela-soares** [comment]
> I did force push to the branch to fix tests, next time I will add commits instead, @rafaela-soares can you please approve the running workflows again? thanks!

Yes, of course, @roi-orca!

Thank you so much for contributing to KICS 🚀

**Comment by roi-orca** [comment]
New commits also request approval for the first time, sorry for that @rafaela-soares 
Next pr time it will not require approval? @rafaela-soares 
Thanks again!

File: assets/queries/ansible/aws/s3_bucket_allows_get_action_from_all_principals/metadata.json
Comment (rafaela-soares) [inline assets/queries/ansible/aws/s3_bucket_allows_get_action_from_all_principals/metadata.json:6]:
```suggestion
  "descriptionText": "S3 Buckets must not allow Get Action From All Principals, as to prevent leaking private information to the entire internet or allow unauthorized data tampering / deletion. This means the 'Effect' must not be 'Allow' when the 'Action' is Get, for all Principals.",
```

File: assets/queries/cloudFormation/aws/iam_password_without_lowercase_letter/metadata.json
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/iam_password_without_lowercase_letter/metadata.json:6]: For this type of query, I suggest something like:

```suggestion
  "descriptionText": "IAM Password should have at least one lowercase letter",
```

(The same applies for the similar queries, like `assets/queries/cloudFormation/aws/iam_password_without_minimum_length`)

File: assets/queries/cloudFormation/aws/iam_policies_with_full_privileges/metadata.json
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/iam_policies_with_full_privileges/metadata.json:6]:
```suggestion
  "descriptionText": "IAM policies shouldn't allow full administrative privileges (for all resources)",
```

File: assets/queries/cloudFormation/aws/iam_policy_grants_assumerole_permission_across_all_services/metadata.json
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/iam_policy_grants_assumerole_permission_across_all_services/metadata.json:6]:
```suggestion
  "descriptionText": "IAM Policy should not grant 'AssumeRole' permission across all services.",
```

File: assets/queries/cloudFormation/aws/s3_bucket_acl_allows_read_or_write_to_all_users/metadata.json
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/s3_bucket_acl_allows_read_or_write_to_all_users/metadata.json:6]: The same typo occurs in `assets/queries/cloudFormation/aws/s3_bucket_acl_allows_read_to_all_users/metadata.json`. Can you also fix it, please? Thank you so much in advance!

File: assets/queries/k8s/containers_with_added_capabilities/metadata.json
Comment (rafaela-soares) [inline assets/queries/k8s/containers_with_added_capabilities/metadata.json:6]:
```suggestion
  "descriptionText": "Containers should not have extra capabilities allowed",
```

File: assets/queries/terraform/aws/ecs_task_definition_network_mode_not_recommended/metadata.json
Comment (rafaela-soares) [inline assets/queries/terraform/aws/ecs_task_definition_network_mode_not_recommended/metadata.json:6]: The same typo (defenition) occurs in `assets/queries/openAPI/2.0/security_definitions_allows_password_flow/metadata.json` and `assets/queries/openAPI/2.0/security_definitions_using_basic_auth/metadata.json`. Can you also fix it, please? Thank you so much in advance!

File: assets/queries/terraform/aws/s3_bucket_allows_get_action_from_all_principals/metadata.json
Comment (rafaela-soares) [inline assets/queries/terraform/aws/s3_bucket_allows_get_action_from_all_principals/metadata.json:6]:
```suggestion
  "descriptionText": "S3 Buckets must not allow Get Action From All Principals, as to prevent leaking private information to the entire internet or allow unauthorized data tampering / deletion. This means the 'Effect' must not be 'Allow' when the 'Action' is Get, for all Principals.",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
Thank you so much for your contribution, @roi-orca ! It really improved the descriptions 🚀 

I only have a few suggestions. Let me know what you think about them, please 😊

**Comment by rafaela-soares** [comment]
> New commits also request approval for the first time, sorry for that @rafaela-soares Next pr time it will not require approval? Thanks again!

No problem, @roi-orca! I approve it again 😊 I think not!

**Comment by roi-orca** [comment]
@rafaela-soares I will commit your suggestions and will align the same queries that were left with the previous `descriptionText` also that are not shown in the pr as changed.
Also will change the descriptions and typos you asked for.
Thanks

**Comment by roi-orca** [comment]
I have a question regarding the `*.md` queries documentation files - should I change the descriptions there as well? or it will automatically update based on the `metadata.json` changes?
Thanks

**Comment by rafaela-soares** [comment]
> @rafaela-soares, I have a question regarding the `*.md` queries documentation files - should I change the descriptions there as well? or it will automatically update based on the `metadata.json` changes? Thanks

Thank you so much for the changes, @roi-orca! It will automatically update by the update-queries-docs action.

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Comment by roi-orca** [comment]
@rafaela-soares I don't have the option to merge this pr. Can you please assist?

**Review by joaoReigota1 — APPROVED** [review]
LGTM

**Comment by rafaela-soares** [comment]
> @rafaela-soares I don't have the option to merge this pr. Can you please assist?

Of course!

The contributors do not have the option to merge. I was waiting for another approval from the team. We will merge it now😊

**Pull request merged**
